compiler/control/OMROptions.cpp

```diff
@@ -3681,6 +3659,28 @@ OMR::Options::jitPostProcess()
    if (_hotMaxStaticPICSlots < 0)
       _hotMaxStaticPICSlots = -_hotMaxStaticPICSlots * _maxStaticPICSlots;
 
+   if (self()->getOption(TR_EnableLargePages))
+      {
+      self()->setOption(TR_EnableLargeCodePages);
+      }
+
+#if defined(TR_HOST_64BIT)
```

Comment:
Yes, you are correct. We are talking about the target here. Fixed in 7a7678e. I have also updated the commit message and the PR description. Thanks for the review!